lib/dynamoid/dirty.rb

```diff
@@ -5,7 +5,7 @@ module Dirty
 
     module ClassMethods
       def from_database(*)
-        super.tap { |d| d.changed_attributes.clear }
+        super.tap { |d| d.send(:clear_changes_information) }
```

Comment:
Probably need to have it handle both versions since older versions of rails I don't think will support this so maybe:

```
d.respond_to?(:clear_changes_information) ? d.clear_changes_information : d.changed_attributes.clear
```

And similar to below to handle both

Reply:
@richardhsu fixed